crates/bevy_core_widgets/src/core_slider.rs

```diff
@@ -72,8 +74,9 @@ pub enum TrackClick {
 )]
 pub struct CoreSlider {
     /// Callback which is called when the slider is dragged or the value is changed via other user
-    /// interaction. If this value is `None`, then the slider will self-update.
-    pub on_change: Option<SystemId<In<f32>>>,
+    /// interaction. If this value is `Callback::Ignore`, then the slider will update it's own
+    /// internal value without notification.
```

Comment:
Can this be like the Checkbox Comments and include the `SliderValue` Component Name, and be above the CoreSlider instead of the callback field.
All comments for these Core* types should preferably be like Checkbox's Comments structure.

Reply:
Basically instead of refering to it as `internal value/state` refer to the Component itself `SliderValue` for example.
Just makes it clear immediately to the user reading it, what the internals will do/use if you don't set this.

Its good either way, just a nit.